README.md

```diff
@@ -1,2 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet) 
 # Weather-App
-Weather-App will show you current weather of any country, was built to improve API and JSON knowladges
+
+Hey, this application you are able to get detailed information about anu country's weather, also we will provide you this country's local time. To create this we have used JS, HTML5, Bootstrap, CSS, Webpack.
+
+
+### Built With
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![BOOTSTRAP](https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white) 
+![JAVASCRIPT](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E")
+
+## Getting Started
+
+## To get local copy up and running follow these simple example steps
+
+```
+git clone https://github.com/AbdumurodovaZulfizar/Weather-App.git
+```
+
+```
+cd Weather-App
+```
+### Prerequisites
+
+- Node
+- Webpack
+
+### Setup
+- run `npm install` to install all required node modules
+- if you want to see live version on your local machine run `npm run build` and open live server
+
+### Live Version
+
+[ToDo List](https://abdumurodovazulfizar.github.io/Weather-App/)
```

Comment:
Oops. Todo List 🤔 , let's correct this.

Reply:
Thanks, it was a little, but ugly mistake 👍